Assets/HoloToolkit-Examples/ColorPicker/GazeableColorPicker.cs

```diff
@@ -50,5 +36,30 @@ void UpdatePickedColor(PickedColorCallback cb)
             Color col = texture.GetPixel((int)pixelUV.x, (int)pixelUV.y);
             cb.Invoke(col);
         }
+
+        public void OnFocusEnter()
+        {
+            gazing = true;
+        }
+
+        public void OnFocusExit()
+        {
+            gazing = false;
+        }
+
+        public void OnInputUp(InputEventData eventData)
```

Comment:
I am OK if it's fixed in a separate one. Although, do you think the up/down might make sense in ISourceState?